lib/renderer.coffee

```diff
@@ -9,7 +9,7 @@ pandocHelper = null # Defer until used
 markdownIt = null # Defer until used
 {scopeForFenceName} = require './extension-helper'
 mathjaxHelper = require './mathjax-helper'
-pathWatcher = require 'pathwatcher-without-runas'
+pathWatcher = require atom.packages.resourcePath + '/node_modules/pathwatcher/lib/main'
```

Comment:
It doesn't work for me with a simple 'pathwatcher'

![screen shot 2015-09-03 at 12 13 54](https://cloud.githubusercontent.com/assets/2906107/9656002/7a50529c-5235-11e5-9618-6d4f2fb23c25.png)